src/main/kotlin/net/atos/client/brp/BrpClientService.kt

```diff
@@ -41,9 +41,17 @@ class BrpClientService @Inject constructor(
         private const val GEBOORTE = "geboorte"
         private const val VERBLIJFPLAATS = "verblijfplaats"
         private const val ADRESSERING = "adressering"
+        private const val INDICATIE_CURATELE_REGISTER = "indicatieCurateleRegister"
 
         private val LOG = Logger.getLogger(BrpClientService::class.java.name)
-        private val FIELDS_PERSOON = listOf(BURGERSERVICENUMMER, GESLACHT, NAAM, GEBOORTE, VERBLIJFPLAATS)
+        private val FIELDS_PERSOON = listOf(
```

Comment:
Is this part of this PR? Does not seem to have to do with BLOKKERING_VANWEGE_VERHUIZING?

Better than to merge this PR on the other indicatie Git branch and not on main. Then you only see the relevant changes here.

Reply:
Yes it is. That's a fix for `ONDER_CURATELE` indications

Reply:
ah right. sorry, missed that